src/functionalTest/java/uk/gov/hmcts/reform/divorce/util/RestUtil.java

```diff
@@ -50,6 +50,25 @@ public static Response putToRestService(String url, Map<String, Object> headers,
         }
     }
 
+    public static Response patchToRestService(String url, Map<String, Object> headers, String requestBody,
+                                     Map<String, Object> params) {
+        if (requestBody != null) {
+            return SerenityRest.given()
+                .headers(headers)
+                .params(params)
+                .body(requestBody)
+                .when()
+                .patch(url)
+                .andReturn();
+        } else {
```

Comment:
this else part is currently not covered by tests

Reply:
Thats what you get when copying and pasting.  I removed the unused else part